server/src/main/java/org/apache/druid/server/coordinator/BalancerStrategy.java

```diff
@@ -55,8 +55,10 @@
   /**
    * Pick the best segment to move from one of the supplied set of servers according to the balancing strategy.
    * @param serverHolders set of historicals to consider for moving segments
-   * @return {@link BalancerSegmentHolder} containing segment to move and server it current resides on
+   * @return {@link BalancerSegmentHolder} containing segment to move and server it current resides on, or null if there
```

Comment:
nit: "server it _currently_ resides on...'

Reply:
Thanks, fixed.